### DISCUSSION TITLE
Is there a way to change the database connection URL conditionally?

### DISCUSSION THREAD

**Discussion by jay2jaykp** [prisma/prisma#13012] [Q&A]
I have a project setup with docker container running two services: PostgreSQL and NodeJS. If I want to connect both with Prisma, the URL connection needs to use the internal port and base service name (i.e. postgres:5432) and if I want to connect the postgres with Prisma Studio from the host machine, the DB connection URL needs to be localhost:5432. But the file `schema.prisma` only takes one connection string. Is there a workaround for such scenario?

**Comment by janpio** [comment]
That is exactly what the environment variable support is meant for: https://www.prisma.io/docs/concepts/components/prisma-schema#accessing-environment-variables-from-the-schema That way you can define one connection URL in your docker container, but another one on the host computer.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
